example/handle_test_result.hpp

```diff
@@ -118,7 +118,7 @@ void handle_test_result(const boost::math::tools::test_result<T>& result,
                        const char* test_name, 
                        const char* group_name)
 {
-#ifdef BOOST_MSVC
+#ifdef _MSC_VER
```

Comment:
There is a potential gotcha here, in that some compilers not msvc define _MSC_VER, hopefully this won't be an issue, but just a heads up...

Reply:
I was under the impression that only MSVC defined `_MSC_VER`. Is there something more robust I should replace this with? Some combination like `defined(_MSC_VER) && defined(_MSVC_LANG)`?